.github/workflows/unit-no-p2.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        java-version: [17]
+        java-version: [17, 21]
```

Comment:
We can just do 21 here.

Reply:
If I get rid of 17 here it means we are no longer running unit tests against Java 17 (just 11 and 21). We need to run unit tests against all three of the latest LTS versions.